WalletWasabi.Gui/ViewModels/MainWindowViewModel.cs

```diff
@@ -111,6 +110,29 @@ public void Initialize()
 			}
 		}
 
+		private void InitializeLockScreen(UiConfig uiConfig)
+		{
+			uiConfig
+				.WhenAnyValue(x => x.LockScreenActive)
+				.Where(x => x)
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(_ => CheckLockScreenType(uiConfig.LockScreenPinHash));
+		}
+
+		private void CheckLockScreenType(string currentHash)
```

Comment:
Rename to CreateLockScreen or just simply move it into the Subscribe.

Reply:
done.